components/CMSUnderConstructionPage.js

```diff
@@ -0,0 +1,57 @@
+import styled from 'styled-components';
+import { formatDate } from '../utility/firebase';
+import nwplusLogo from '../assets/nwplus.svg';
+
+const NwplusImage = styled.img`
+  width: 150px;
+  height: 150px;
+`;
+
+const Container = styled.div`
+  width: 100vw;
+  height: 100vh;
+`;
+
+const MainContent = styled.div`
+  width: 400px;
+  height: 400px;
+  font-family: 'HK Grotesk';
+  font-size: 24px;
+  text-align: center;
+
+  position: absolute;
```

Comment:
i enjoy flexbox but this is okay too

Reply:
How would you do it with flexbox out of curiosity?

Reply:
justify-content and align-content i think